Typescript problems

Getting this with React-Markdown 4.3.0 and Typescript 3.7.4.

Not sure what additional detail I could help with here :/

```
ERROR in node_modules/react-markdown/index.d.ts
ERROR in node_modules/react-markdown/index.d.ts(138,16):
TS2665: Invalid module name in augmentation. Module 'react-markdown/with-html' resolves to an untyped module at 'node_modules/react-markdown/with-html.js', which cannot be augmented.

ERROR in node_modules/react-markdown/index.d.ts
ERROR in node_modules/react-markdown/index.d.ts(140,35):
TS2307: Cannot find module 'react-markdown'.
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -135,9 +135,3 @@ declare namespace ReactMarkdown {
 
 export = ReactMarkdown
 
-declare module 'react-markdown/with-html' {
-    import {Component} from 'react'
-    import ReactMarkdownRoot from 'react-markdown'
-
-    export default class ReactMarkdown extends Component<ReactMarkdownRoot.ReactMarkdownProps, {}> {}
-}

--- a/with-html.d.ts
+++ b/with-html.d.ts
@@ -1,0 +1,5 @@
+import {Component} from 'react'
+import ReactMarkdownRoot from 'react-markdown'
+
+export default class ReactMarkdown extends Component<ReactMarkdownRoot.ReactMarkdownProps, {}> {}
+

--- a/with-html.d.ts
+++ b/with-html.d.ts
@@ -1,5 +1,5 @@
 import {Component} from 'react'
-import ReactMarkdownRoot from 'react-markdown'
+import ReactMarkdownRoot = require('react-markdown')
 
-export default class ReactMarkdown extends Component<ReactMarkdownRoot.ReactMarkdownProps, {}> {}
-
+declare class ReactMarkdown extends Component<ReactMarkdownRoot.ReactMarkdownProps, {}> {}
+export = ReactMarkdown

